src/05-invoicing/invoices.apib

```diff
@@ -20,6 +20,8 @@ Get a list of invoices.
                     + draft
                     + outstanding
                     + matched
+            + date_before: `2016-02-04` (string, optional)
+            + date_after: `2016-02-04` (string, optional)
```

Comment:
What is the property we are filtering on here? `date_before` is ambiguous: it could be `invoice_date`, `created_at` as well as `updated_at`.

We can make the filter name resemble or reference the actual property to filter on, as to make it as obvious/easy as possible to integrators. I don't think specifying it only in the changelog is clear enough.